typings/index.d.ts

```diff
@@ -326,6 +340,17 @@ declare module 'discord.js' {
 
   type AllowedImageFormat = 'webp' | 'png' | 'jpg' | 'jpeg' | 'gif';
 
+  enum ChannelTypes {
+    TEXT = 0,
+    DM = 1,
+    VOICE = 2,
+    GROUP = 3,
+    CATEGORY = 4,
+    NEWS = 5,
+    STORE = 6,
+    STAGE = 13,
+  }
+
```

Comment:
This should be moved to the enums at the top of the file because it's only exported in `Constants` and not as a direct export